test/algorithm/copy.cpp

```diff
@@ -67,5 +77,9 @@ int main()
         CHECK(std::strcmp(sz, buf) == 0);
     }
 
+#if RANGES_CXX_CONSTEXPR >= RANGES_CXX_CONSTEXPR_14
+    static_assert(test_constexpr_copy(), "");
+#endif
```

Comment:
...and then move these two lines to line 29. (i.e., keep the `constexpr` test together as a single unit.)

Reply:
done